ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -1196,7 +1196,20 @@ device_classes:
                     service:
                         type: COMMAND
                         usessh: true
-                        commandTemplate: "/bin/sh -c 'export PATH=$$PATH:/bin:/sbin:/usr/bin:/usr/sbin; if command -v systemctl > /dev/null 2>&1; then echo \"SYSTEMD\"; sudo systemctl list-units -t service --all --no-page --no-legend | sed /not-found/d | cut -d\" \" -f1 | xargs -n 1 sudo systemctl status -l -n 0 2>&1 || true; elif command -v service >/dev/null 2>&1; then echo \"SYSTEMV\"; sudo service --status-all 2>&1 || true; else echo \"UNKNOWN\"; exit 127; fi'"
+                        # Check for systemd, upstart and systemv for services
+                        commandTemplate: >
+                                           /bin/sh -c 'export PATH=$$PATH:/bin:/sbin:/usr/bin:/usr/sbin;
+                                           if command -v systemctl > /dev/null 2>&1;
+                                           then echo "SYSTEMD";
+                                           sudo systemctl list-units -t service --all --no-page --no-legend |
+                                           sed /not-found/d | cut -d" " -f1 | xargs -n 1 sudo systemctl status -l -n 0 2>&1 || true;
```

Comment:
We can eliminate one pipe and one cut but combinind 2 sed commands.

So instead of `sed /not-found/d | cut -d" " -f1 ` you can write: `sed '/not-found/d ; s/\s.*$//'`

This assumes that all the "bad stuff" you want to remove follows the first whitespace character, and will also handle cases where that whitespace is not just a blank char so can be more robust.